packages/bridge-ui/vite.config.ts.timestamp-1671142216308.mjs

```diff
@@ -0,0 +1,26 @@
+// vite.config.ts
```

Comment:
should this file be gitignored?

Reply:
@cyberhorsey I am not sure about this, could you check?

Reply:
How did you generate this file? Did you run build locally or something? This shouldn't be pushed. I think you can add `vite.config.ts.timestamp-*.mjs` to gitignore.

Reply:
added to gitignore